src/fu-self-test.c

```diff
@@ -693,7 +693,7 @@ fu_pending_func (void)
 	g_object_unref (device);
 
 	/* get device that does not exist */
-	device = fu_pending_get_device (pending, "self-test", &error);
+	device = fu_pending_get_device (pending, "2ba16d10df45823dd4494ff10a0bfccfef512c9d", &error);
```

Comment:
Some of these it's no longer obvious what you're testing for.  Can you add a comment with /* self-test */ or something when using the sha1 in code?

Reply:
Fixed, thanks.